frontend/src/views/Bridge.vue

```diff
@@ -451,6 +524,9 @@ export default Vue.extend({
       return fromWeiEther(wei);
     },
     async transferToStorage(){
+      // chars & weapon ids are ints; other nfts are <type.id>
+      if(this.nftType !== 'weapon' && this.nftType !== 'character') this.selectedNftId = this.selectedNftId.split('.')[1];
```

Comment:
I'll look into it a bit, but maybe it's possible to map the ids if they are different? Also, that's bridge-specific, right? Bcs I'm using fetching weapons/junk/shields and all that by ids and it's all done in a similar manner, not some kind of type.id. I'll look into it and come back to you